File: PlaylistService/PlaylistService/Common/Entities/Playlist.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;

namespace PlaylistService.Common.Entities
{
    public class Playlist
    {
        [BsonId]
        public string Id { get; set; } = ObjectId.GenerateNewId().ToString();
        public string Name { get; set; } = String.Empty;
        public string UserId { get; set; } = String.Empty;
        public List<string> SongIds { get; set; } = new List<string>();
    }
}

File: AIOrchestraUI/src/app/models/playlist.ts
import { Song } from "./song";

export class Playlist {
    Id : string;
    Name : string;
    UserId : string;
    SongIds: string[];
    Songs : Song[];

    constructor(id: string, name: string, userId: string, songs: string[]) {
        this.Id = id;
        this.Name = name;
        this.UserId = userId;
        this.SongIds = songs;
        this.Songs = [];
    }
}